frontend/mr-admin-flate/src/pages/notifikasjoner/NotifikasjonerPage.module.scss

```diff
@@ -5,6 +5,7 @@
 
 .container {
   margin-left: 4rem;
+  max-width: 704px;
```

Comment:
Hva er dette magiske 704-tallet? Klarer vi å ikke være så konkrete? Det skaper ingen god responsiv opplevelse.

Reply:
Har det fra skissene. Hva er det vi pleier å gjøre for å ha en max bredde på ting?

Reply:
Jeg ville heller prøvd med prosent, så det flyter opp og ned basert på skjermstørrelsen. Feks. noe slikt 
```
.container {
  max-width: 100%; // Default 100%
  @media screen and (min-width: 850px) { // Når skjermen blir større enn 850 px så er det mindre max-width.
    max-width: 35%;
  }
}
```